integration-tests/jest-puppeteer.config.js

```diff
@@ -6,6 +6,6 @@ module.exports = {
     slowMo: process.env.SLOW === 'true' ? 250 : undefined,
     defaultViewport: { width: 1200, height: 800 },
     protocolTimeout: 720_000, // 12 minutes
-    args: [],
+    args: ['--flag-switches-begin', '--disable-features=TrackingProtection3pcd', '--flag-switches-end'],
```

Comment:
The puppeteer upgrade brings with it an upgrade to the version of Chrome used for integration tests. The Chrome upgrade blocks third-party cookies, which causes the `run-rstudio` and `run-analysis` tests to fail.

This change tells puppeteer's Chrome to allow third-party cookies.